packages-tests/Caching/Config/FileHashComputer/FileHashComputerTest.php

```diff
@@ -19,9 +18,10 @@ protected function setUp(): void
         $this->fileHashComputer = $this->make(FileHashComputer::class);
     }
 
-    #[RunInSeparateProcess]
     public function testRectorPhpChanged(): void
     {
+        SimpleParameterProvider::setParameter(Option::REGISTERED_RECTOR_RULES, null);
```

Comment:
here the tweak, clean up parameters before testing to avoid overlapped.

Reply:
Should happen in abstract test class

Reply:
No, This tests bootFromConfigFiles() so it need immediatelly resetted here, and this run in every after calls, per test has 2 calls bootFromConfigFiles() to compare before vs after